### DISCUSSION TITLE
Does the photo taking function work for iOS Safari?

### DISCUSSION THREAD

**Discussion by Vasheer26** [google/model-viewer#2952] [Q&A]
Hi everyone.

Can anyone confirm the photo taking button working correctly on iOS safari?

When clicking to take photo, only the 3D object is rendered.  
The environment is not captured and is rendered as black.
https://ibb.co/BcYCctY

I am using iOS15.
For Android Scene Viewer, the photo button takes screenshot fine.

Anyone have a solution?

Thank you very much.

**Comment by elalish** [comment]
Wow,  Safari 15 has introduced a lot  of bugs in Quick Look apparently. Please file this on WebKit (Safari): https://bugs.webkit.org/

**Comment by Vasheer26** [comment]
Just to give everyone an update.  
Potentially, this might be an issue only on my device which is an iPhone X iOS15.
I had a friend test the same thing with his iPhone XS iOS15 and it was working correctly.

**Reply by ghost** [reply]
Just updated an iPad Pro to 15.1; taking photos in Quicklook is broken for me too, I actually just get a black image; I don't even see the 3D model.

**Comment by iamkim17** [comment]
Any update on this? I tried it on iPhone 8 with iOS15 but still, the background is black but the 3D is rendered when clicking take photo.

**Reply by elalish** [reply]
Again, Webkit is the place to track Apple's bugs.

**Comment by elalish** [comment]
@grorg Not sure if this has been fixed yet?
